frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.unit.spec.tsx

```diff
@@ -0,0 +1,119 @@
+import { renderWithProviders, screen } from "__support__/ui";
+import type { EmbeddingEntityType } from "metabase/embedding-sdk/store";
+import * as domUtils from "metabase/lib/dom";
+import {
+  createMockEmbedOptions,
+  createMockEmbedState,
+  createMockState,
+} from "metabase-types/store/mocks";
+
+import { BrowseNavSection } from "./BrowseNavSection";
+
+describe("BrowseNavSection", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("should render Models, Databases, and Metrics when not embedding in iframe", () => {
```

Comment:
Add an additional case for non-embedding, every item should be visible (with proper permissions)